skulpt.py

```diff
@@ -225,7 +201,28 @@ def test(debug_mode=False, p3=False):
     ret4 = 0
     if not p3:
         buildNamedTestsFile()
-        ret1 = os.system("{0} {1} {2} -- {3}".format(jsengine, ' '.join(getFileList(FILE_TYPE_TEST)), ' '.join(TestFiles), debugon))
+
+        # Initial "requires"
```

Comment:
can we store the require.js file instead of building it everytime?

Reply:
Yes.  If we're serious about doing this, I can clean it up a bit.